app/assets/stylesheets/forever_style_guide/modules/_modal.scss

```diff
@@ -0,0 +1,100 @@
+$modal-padding-footer: 15px;
+$modal-form-width-lg: 355px;
+$modal-product-suggestion-sm: 170px;
+
+.modal-close-position {
+  float: right;
+}
+
+.modal-theme {
+  color: $color-white;
+}
+
+.modal-title {
+  color: inherit;
+  font-family: $font-face-light;
+}
+
+.modal-footer-buttons {
+  padding: $modal-padding-footer;
+  border-top: 1px solid $color-gray-400;
+  text-align: right;
```

Comment:
text-align should be precede border-top in declaration order since it's a typography property http://codeguide.co/#css-declaration-order